test/helpers/story/drivers/dapp-utils/yearn.utils.ts

```diff
@@ -0,0 +1,65 @@
+import chai, { expect } from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+
+import { LoanHelpersReturn } from "../../../loans"
+import LoanStoryTestDriver from '../loan-story-test-driver'
+chai.should()
+chai.use(solidity)
+
+async function lendYearn(
+    hre: HardhatRuntimeEnvironment,
+    loan: LoanHelpersReturn
+  ): Promise<void> {
+    const { details, diamond } = loan
+    await diamond
+      .connect(details.borrower.signer)
+      .yearnDeposit(
+        details.loan.id,
+        details.loan.lendingToken,
+        details.loan.borrowedAmount
+      )
+
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+    const daiBalance = await details.lendingToken.balanceOf(escrowAddress)
+    daiBalance.eq(details.loan.borrowedAmount).should.eql(true, '')
```

Comment:
@dumebi could this be easierly read by saying `dailBalance.should.eq(details.loan.borrowedAmount)` (idk if this is what this line is trying to say tbh lol) 😝

Reply:
yes, you're right